app/src/main/java/hr/foi/air/servicesync/business/ReservationManager.kt

```diff
@@ -2,7 +2,7 @@ package hr.foi.air.servicesync.business
 
 import hr.foi.air.servicesync.backend.FirestoreService
 
-class ReservationManager(private val firestoreService: FirestoreService) {
```

Comment:
Maybe consider leaving as private so it can't be accessed from outside. UI should not be allowed to acces this property.

Reply:
Everything was tested and the conclusion was it needed to be public.